include/ppx/xr_component.h

```diff
@@ -94,12 +101,16 @@ class XrComponent
     {
         if (mConfigViews.empty())
             return 0;
+        if (mCreateInfo.resolution.width > 0)
```

Comment:
shouldn't that be returned only while there are no view configured?
(Not familiar with how XR manages view, but I'd guess XR takes the size hint, but might return a view with a different size once created no?)

Reply:
oh ok, thanks